test/test_ops.py

```diff
@@ -200,7 +200,7 @@ def test_multiple_devices(self, devices, dtype, op):
     #   values on the tensors from sample_inputs func for the corresponding op.
     @onlyNativeDeviceTypes
     @suppress_warnings
-    @ops(_ref_test_ops, allowed_dtypes=(torch.float32, torch.long, torch.complex64))
+    @ops(_ref_test_ops, allowed_dtypes=(torch.float64, torch.long, torch.complex128))
```

Comment:
This change makes sense give how often NumPy promotes to double, but it's also a big change.

How much longer does this test take to run than when it uses float32 and complex64?

Reply:
It's actually faster on my computer with the proposed change. It took 63.45 seconds with float32 and complex64 and 62.10 seconds with float64 and complex128.

```
python -m pytest test/test_ops.py -k 'test_reference_testing'

# float32 and complex64
1 failed, 180 passed, 91 skipped, 32563 deselected, 2 warnings in 63.45s (0:01:03)

# float64 and complex128:
181 passed, 91 skipped, 32563 deselected, 2 warnings in 62.10s (0:01:02)
```

Reply:
Now how the heck is that even possible.

But OK I suppose my concern is addressed.